src/global/types.h

```diff
@@ -2094,4 +2105,10 @@ typedef enum INPUT_LAYOUT {
     INPUT_LAYOUT_NUMBER_OF,
 } INPUT_LAYOUT;
 
+typedef enum BUTTON_TYPE {
+    BUTTON = 0,
+    AXIS = 1,
```

Comment:
Please use BT_ prefix

Reply:
Oof good call. Idk how I forgot this prefix. That's why I used `axs` instead of axis. I wanted to avoid overlapping the same name.